tips/tips-pl.txt

```diff
@@ -0,0 +1,110 @@
+Tipster
+Witaj w Tipster! Kliknij w ikon� po lewej stronie wskaz�wki, �eby uzyska� informacje. Kliknij w tekst, �eby przej�� do nast�pnej wskaz�wki.
+http://www.github.com/HaikuArchives/Tipster
+%
+Terminal
+Otw�rz plik, �cie�k�, lub URL w Terminalu, klikaj�c na niego i przytrzymuj�c klawisz ALT.
+https://www.haiku-os.org/docs/userguide/en/applications/terminal.html#shortcuts
+%
+GUI
+Uk�adaj i ��cz okna, przytrzymuj�c klawisz OPT i przesuwaj�c ich kraw�dzie lub tytu�y okienek jedn� obok drugiej.
+https://www.haiku-os.org/docs/userguide/en/gui.html#move-resize
+%
+GUI
+Przytrzymuj�c klawisze ALT + CTRL mo�esz wygodnie przesuwa� okna (lewy przycisk myszy) i zmienia� ich rozmiar (prawy przycisk myszy).
+https://www.haiku-os.org/docs/userguide/en/gui.html#move-resize
+%
+GUI
+U�yj Workspaces - wirtualnych pulpit�w - �eby podzieli� twoj� prac� a� do 32 przestrzeni roboczych.
+https://www.haiku-os.org/docs/userguide/en/workspaces.html
+%
+GUI
+Klikaj�c prawym przyciskiem na zak�adk� okna (lub gdziekolwiek przytrzymuj�c klawisze ALT + CTRL), okno przeniesie si� do ty�u, a klikaj�c dwa razy lewym przyciskiem myszy, mo�emy zminimalizowa� okno do paska zada�.
+https://www.haiku-os.org/docs/userguide/en/gui.html
+%
+Preferencje
+Usprawnij swoj� prac� tworz�c w�asne skr�ty klawiszowe w Preferencjach skr�t�w klawiszowych. (Shortcut Preferences)
+https://www.haiku-os.org/docs/userguide/en/keyboard-shortcuts.html
+%
+Aplikacja
+Ustaw plik dzwi�kowy dla "Vision Nick Notification" w ustawieniach d�wi�ku aby otrzymywa� powiadomienie za ka�dym razem kiedy kto� napisze twoje imi� w kliencie IRC.
+https://www.haiku-os.org/docs/userguide/en/applications/vision.html
+%
+Aplikacja
+Przycisk TAB doka�cza komendy i �cie�ki w Terminalu oraz nicki w aplikacji Vision. Po prostu zacznij pisa� i u�yj klawisza TAB aby dopasowa�.
+%
+R�ne
+�r�d�o systemu Haiku jest dobrym miejscem przyk�ad�w u�ycia API Haiku. Przeszukaj go dla swojej ulubionej klasy.
+http://xref.plausible.coop/source/
+%
+R�ne
+Szybko naci�nij CTRL + TAB, �eby prze��czy� si� pomi�dzy dwiema aplikacjami. Przytrzymaj d�u�ej t� kombinacj� klawiszy, �eby zobaczy� wszystkie dzia�aj�ce aplikacje w Twitcher.
+https://www.haiku-os.org/docs/userguide/en/twitcher.html
+%
+Preferencje
+Dzi�ki filtrowaniu z wyprzedzeniem Tracker wy�wietla tylko te pliki, kt�rymi jeste� zainteresowany.
+https://www.haiku-os.org/docs/userguide/en/tracker.html#tracker-preferences
+%
+Preferencje
+Je�li uwa�asz, �e wolisz korzysta� z gorszego uk�adu klawiszy skr�t�w Linux/Windows (ALT <-> CTRL) mo�esz zmeni� to i par� innych rzeczy w Preferencjach uk�adu klawiszy. (Keymap preferences)
+https://www.haiku-os.org/docs/userguide/en/preferences/keymap.html
+%
+Terminal
+Upuszczaj�c plik do Terminala, mo�na wstawi� do niego jego �cie�k�. Robi�c to przy u�yciu prawego przycisku myszy mamy do dyspozycji jeszcze inne u�yteczne czynno�ci. 
```

Comment:
Upuszczenie pliku w Terminalu wstawia jego pełną ścieżkę, to samo przy użyciu prawego przycisku myszy daje do dyspozycje jeszcze inne użyteczne czynności.